LICENSE

```diff
@@ -330,7 +282,7 @@ APPENDIX B: Additional licenses relevant to this product:
 
     Code locations:
     -------------------------------------------------------------    
-    This product contains code for encoding, decoding and testing UTF8:
```

Comment:
This was a typo, that I just discovered. Zero-allocation-hashing is used for xxHash testing, not UTF8.